modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ViewObjectDefinitions/objectDefinitionUtil.ts

```diff
@@ -51,6 +73,134 @@ export async function deleteObjectDefinition(
 	});
 }
 
+export async function deleteObjectDefinition(
```

Comment:
@igor-franca Please, also declare the parameters of this function as named parameters.